src/polyfills/details/_base.scss

```diff
@@ -6,6 +6,10 @@
 summary {
 	// Make sure summary remains visible
 	display: list-item !important;
+	list: {
+		style: none;
```

Comment:
@duboisp What you linked is meant for use in Ruby/Rails configuration options that are set outside of SCSS files. This code block is situated in an SCSS file and thus is part of a totally different kind of context that has nothing to do with those options.

On this specific line, I'm using a nested ``style`` property within the previous line's the ``list`` namespace, which will generate a ``list-style`` property. That's a proper use of [SASS' nested properties extension](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested_properties).